OneSignalExample/Assets/OneSignal/Editor/PostProcessBuildPlayer_iOS.cs

```diff
@@ -67,14 +65,35 @@ public static void OnPostProcessBuild(BuildTarget target, string path)
       FileUtil.CopyFileOrDirectory ("Assets/OneSignal/Platforms/iOS/NotificationService.h", path + "/OneSignalNotificationExtensionService/NotificationService.h");
       FileUtil.CopyFileOrDirectory ("Assets/OneSignal/Platforms/iOS/NotificationService.m", path + "/OneSignalNotificationExtensionService/NotificationService.m");
 
-      project.WriteToFile (projPath);
-
-      string projectFile = path + "/Unity-iPhone.xcodeproj/project.pbxproj";
-      string contents = File.ReadAllText(projectFile);
+      project.WriteToFile (projectPath);
 
+      string contents = File.ReadAllText(projectPath);
       InsertStaticFrameworkIntoTargetBuildPhaseFrameworks("libOneSignal", "CD84C25F20742FAB0035D524", notificationServiceTarget, ref contents, project);
 
-      File.WriteAllText(projectFile, contents);
+      project.ReadFromString(contents);
+
+
+      // enable the Notifications capability
+      File.WriteAllText(projectPath, contents);
+      var entitlementPath = path + "/" + targetName + "/" + targetName + ".entitlements";
+      
+      PlistDocument entitlements = new PlistDocument();
+      entitlements.root.SetString("aps-environment", "development");
```

Comment:
Will using `"development"` here create any issues when building a release .IPA? Or will Xcode automatic replace this to development when they do so?